pkg/ocm/sca/architectures_gather.go

```diff
@@ -20,6 +23,17 @@ var kubernetesArchMapping = map[string]string{
 	"arm64":   "aarch64",
 }
 
+type ClusterArchitecture struct {
```

Comment:
This type can be lowercase too or?

Reply:
Yes, it is only used in this package/file